runtime/appruntime/metrics/gcp/cloud_monitoring.go

```diff
@@ -109,7 +119,11 @@ func (x *Exporter) getMetricData(newCounterStart, endTime time.Time, collected [
 		}
 
 		svcNum := m.Info.SvcNum()
-		metricType := "custom.googleapis.com/" + m.Info.Name()
+		encoreCloudMetricName, ok := x.encoreCloudMetricNames[m.Info.Name()]
+		if !ok {
+			continue
```

Comment:
Is this expected to happen, or is it guarding against something that never should happen unless something is wrong? If the latter it would be useful to log an error like we do above

Reply:
This should never happen. I'll add a log line 👍